app/javascript/controllers/feedback_controller.js

```diff
@@ -0,0 +1,14 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ['container']
+
+  toggle(e) {
+    e.preventDefault()
+    this.containerTarget.classList.toggle('open')
+  }
+
+  reset(e) {
```

Comment:
Should this be `cancel` instead to match the action name in the template: https://github.com/sul-dlss/stanford-arclight/pull/513/files#diff-90f3b4ef8af0fe2f5ff19d1c628b8f58e6d26770854922c2492b2e5f8e096187R50

Reply:
Yes. Thank you for catching this detail!